File: Facultate/Anul 2/Sem 2/MPP/Laburi/CSharpLab_3/Shared/dtos/MakeReservationDTO.cs
namespace Shared.dtos;

public class MakeReservationDTO
{
    public long RideId { get; set; }
    public string ClientName { get; set; }
    public int Seats { get; set; }

    public MakeReservationDTO(long rideId, string clientName, int seats)
    {
        RideId = rideId;
        ClientName = clientName;
        Seats = seats;
    }
}

File: Facultate/Anul 2/Sem 2/MPP/Laburi/Edi/JavaClient C# Server/Shared/src/main/java/dtos/MakeReservationDTO.java
package dtos;

import java.io.Serializable;
import com.google.gson.annotations.SerializedName;

public class MakeReservationDTO implements Serializable {

    @SerializedName("RideId")
    private final Long rideId;
    @SerializedName("ClientName")
    private final String clientName;
    @SerializedName("Seats")
    private final int seats;

    public MakeReservationDTO(Long rideId, String clientName, int seats) {
        this.rideId = rideId;
        this.clientName = clientName;
        this.seats = seats;
    }

    public Long getRideId() {
        return rideId;
    }
    public String getClientName() {
        return clientName;
    }
    public int getSeats() {
        return seats;
    }
}
